QRCoder/PngByteQRCode.cs

```diff
@@ -261,12 +261,7 @@ public void WriteScanlines(byte[] scanlines)
             _stream.WriteByte(0x9C); // Check bits.
 
             // Compressed data.
-            idatStream.Position = 0;
-#if NET35
             idatStream.WriteTo(_stream);
```

Comment:
WriteTo writes the entire MemoryStream (disregarding position) to the target stream, and is a supported method across all .NET variations.